### PR TITLE
Henda52 patch 1 hej

### PR DESCRIPTION
**Pull request by henda52**
kasdk

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,9 @@
 testar samarbete
 
 my first commit
+
+
+hej hlksal
+
+
+hej
```
Comment (crsh) [inline README.md:11]: In the interest of brevity, I might omit this.

File: README.md [hunk 2]
```diff
@@ -2,3 +2,9 @@
 testar samarbete
```
Comment (crsh) [inline README.md]: Interesting thought! 🤔

**Review by crsh — CHANGES_REQUESTED** [review]
Great changes. Maybe just add another "hej" to the end of the file.

File: README.md (hunk 2 above)
Comment (henda52) [reply README.md]: added Hej

**Pull request merged**